plugin/megaraid/megaraid.py

```diff
@@ -1181,8 +1200,9 @@ def volume_read_cache_policy_update(self, volume, rcp,
         storcli always enable read cache and no way to change it
         """
         raise LsmError(ErrorNumber.NO_SUPPORT,
-                       "LSI MegaRAID always enable read cache and refused to "
-                       "change that.")
+                       "MegaRAID always enable read cache and refused to "
+                       "change that. You can change read ahead cache "
+                       "setting via storcli/perccli")
```

Comment:
The plugin doesn't advertise that is supports the method `volume_read_cache_policy_update` so technically this method should never be called by a client, but I guess it's OK to have it.